src/partials/home/companies.js

```diff
@@ -12,74 +9,69 @@ import AIRBNB from "../../../static/logos/logo-airbnb.svg";
-            style={{ maxWidth: "300px", filter: "invert(1)" }}
-            src={FIVETHIRTYEIGHT}
-          />
-          <img
-            className="Companies-logo"
-            style={{
-              maxWidth: "177px",
-              paddingTop: "2px"
-            }}
-            src={REDFIN}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "350px" }}
-            src={USAFACTS}
-          />
-        </ul>
-        <ul className="Companies-List">
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "200px", filter: "invert(1)" }}
-            src={VIACOM}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "300px", height: "50px" }}
-            src={BENAROYA}
-          />
-          <img
-            className="Companies-logo"
-            style={{
-              maxWidth: "190px",
-              paddingBottom: "3px",
-              filter: "invert(1)"
-            }}
-            src={ZILLOW}
-          />
-          <img
-            className="Companies-logo"
-            style={{ maxWidth: "141px", filter: "invert(1)" }}
-            src={TUNE}
-          />
-        </ul>
-        <p className="SubHeading">
-          <Link to="/about#showcase" className="isLight">
-            See showcase&nbsp;
-            <Icon glyph="internal-link" style={{ filter: "invert(1)" }} />
-          </Link>
-        </p>
-      </div>
-    );
+const SectionHeading = styled.h2`
+  color: ${({ theme }) => theme.color.darkBrown};
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 1.58;
+  letter-spacing: 0.48px;
+  margin: 0;
+  text-align: center;
+  @media ${({ theme }) => theme.mediaQuery.md} {
+    font-size: 24px;
   }
-}
+`;
+
+const CompaniesList = styled.ul`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-gap: 4rem;
+  margin: 5rem 0;
+  @media ${({ theme }) => theme.mediaQuery.sm} {
+    grid-template-columns: repeat(4, 1fr);
+    grid-auto-rows: 1fr;
+    grid-gap: 3rem;
+  }
+`;
+
+const CompanyLogo = styled.img`
+  align-self: center;
+  justify-self: center;
+  max-height: 100px;
+  max-width: 200px;
+  width: 100%;
+`;
+
+const Companies = () => (
+  <LandingSectionWrapper bg={importedTheme.color.lightGray}>
+    <LandingSectionContent>
+      <SectionHeading>A Few of Our Fans</SectionHeading>
+      <CompaniesList>
+        <CompanyLogo src={AIRBNB} />
+        <CompanyLogo src={FIVETHIRTYEIGHT} />
+        <CompanyLogo src={REDFIN} />
+        <CompanyLogo src={USAFACTS} />
+        <CompanyLogo src={VIACOM} />
+        <CompanyLogo src={BENAROYA} />
+        <CompanyLogo src={ZILLOW} />
+        <CompanyLogo src={TUNE} />
+      </CompaniesList>
+      <LinkButton to="/about#showcase" color={importedTheme.color.black}>
+        SEE SHOWCASE
+      </LinkButton>
+    </LandingSectionContent>
+  </LandingSectionWrapper>
+);
 
 Companies.propTypes = {
```

Comment:
maybe i'm blind but i don't see a prop `style` in use anymore here, could these be removed?